src/neuroglancer/datasource/dvid/backend.ts

```diff
@@ -37,13 +52,34 @@ class VolumeChunkSource extends ParameterizedVolumeChunkSource<VolumeChunkSource
       // computeChunkBounds.
       let chunkPosition = this.computeChunkBounds(chunk);
       let chunkDataSize = chunk.chunkDataSize!;
-      path = `/api/node/${params['nodeKey']}/${params['dataInstanceKey']}/raw/0_1_2/` +
-          `${chunkDataSize[0]}_${chunkDataSize[1]}_${chunkDataSize[2]}/` +
-          `${chunkPosition[0]}_${chunkPosition[1]}_${chunkPosition[2]}/nd`;
+
+      // if the volume is an image, get a jpeg
+      path = this.getPath(chunkPosition, chunkDataSize, params);
     }
+    let decoder = this.getDecoder(params);
     return sendHttpRequest(
                openShardedHttpRequest(params.baseUrls, path), 'arraybuffer', cancellationToken)
-        .then(response => decodeRawChunk(chunk, response));
+        .then(response => decoder(chunk, response));
+  }
+  getPath(chunkPosition: Float32Array, chunkDataSize: Float32Array, params: any) {
+    if (params.volumeType === VolumeType.IMAGE) {
+      return `/api/node/${params['nodeKey']}/${params['dataInstanceKey']}/raw/0_1_2/` +
+          `${chunkDataSize[0]}_${chunkDataSize[1]}_${chunkDataSize[2]}/` +
+          `${chunkPosition[0]}_${chunkPosition[1]}_${chunkPosition[2]}/jpeg`;
+    } else {
+      // volumeType is SEGMENTATION
+      return `/api/node/${params['nodeKey']}/${params['dataInstanceKey']}/raw/0_1_2/` +
+          `${chunkDataSize[0]}_${chunkDataSize[1]}_${chunkDataSize[2]}/` +
+          `${chunkPosition[0]}_${chunkPosition[1]}_${chunkPosition[2]}?compression=googlegzip`;
+    }
+  }
+  getDecoder(params: any) {
+    if (params.volumeType === VolumeType.IMAGE) {
```

Comment:
You can determine this based on this.spec.compressedSegmentationBlockSize instead (or an explicit encoding parameter if you want to go that route).  No need to pass params, since they can be accessed as this.parameters.